.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -49,6 +49,15 @@ dynamodb_CreateTable:
             - description:
               snippet_tags:
                 - dynamodb.java2.create_table.main
+    PHP:
+      versions:
+        - sdk_version: 3
+          github: php/example_code/dynamodb/dynamodb_basics

```

Comment:
Link to the folder that has the README (up one level in this case).

Reply:
Fixed